main/main.cpp

```diff
@@ -621,6 +621,9 @@ void Main::print_help(const char *p_binary) {
 	print_help_title("Debug options");
 	print_help_option("-d, --debug", "Debug (local stdout debugger).\n");
 	print_help_option("-b, --breakpoints", "Breakpoint list as source::line comma-separated pairs, no spaces (use %%20 instead).\n");
+#ifdef DEBUG_ENABLED
+	print_help_option("--fail-on-error", "Exit with a failure code if any errors are encountered.\n", CLI_OPTION_AVAILABILITY_TEMPLATE_DEBUG);
```

Comment:
Make the description clearer. Right now, it can be understood as the engine exiting as soon as an error is encountered.

```suggestion
	print_help_option("--fail-on-error", "Set the exit code to 1 if any errors are encountered.\n", CLI_OPTION_AVAILABILITY_TEMPLATE_DEBUG);
```

Reply:
Good point.